src/pages/DistributionsPage/AllocationsTable.tsx

```diff
@@ -5,9 +5,9 @@ import uniqBy from 'lodash/uniqBy';
 
 import { Paginator } from '../../components/Paginator';
 import { DISTRIBUTION_TYPE } from '../../config/constants';
-import { NewApeAvatar, makeTable } from 'components';
-import { Flex, Text, Panel, Button, Link } from 'ui';
-import { smartRounding, numberWithCommas, shortenAddress } from 'utils';
```

Comment:
`smartRounding` shouldn't have been removed here

Reply:
<img width="1440" alt="Screen Shot 2022-09-23 at 6 18 16 AM" src="https://user-images.githubusercontent.com/94828675/191891350-6fbaa4e0-9aaa-427f-aaac-199fd755153f.png">

are we sure it should stay?

Reply:
your IDE might just be out of date -- try opening the `utils` file and see if the error goes away